### ISSUE TITLE
update intergenic logic

### ISSUE THREAD

**Issue by jharenza** [d3b-center/annoFuse#54]
Seeing instances of `intergenic` gene fusions being called `intragenic` in V16:

```
BS_03FT4S8B: LINC01019--LINC01019/IRX1
BS_7ZV3DPGT: CNTN6--CNTN6/RPL23AP38
``` 

Comparing the `intergenic` annotations to these, it seems that these are being called `intragenic` due to the fact that Gene1A and Gene1B are identical. 

If all intergenic fusions will always have a `/`, then I suggest simple logic built around that.

**Comment by kgaonkar6** [comment]
oh yeah I'm doing these cases, I'll update that to make the above intergenic fusions
https://github.com/d3b-center/annoFuse/blob/593d031f667fd803cbc1e7bd93f12da548e00990/R/fusion_standardization.R#L123-L126
      BreakpointLocation = case_when(
        Gene1A==Gene1B ~ "Intragenic",
        grepl("/",FusionName) ~ "Intergenic",
        TRUE ~ "Genic"),

**Comment by jharenza** [comment]
closed with #56
